features/google/src/views/overview/overview.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from "react";
+import { GoogleContext } from "../../context/google-context.jsx";
+
+const Overview = () => {
+    const { handleSelectFile, files, handleRemoveFile, googleAccount } =
+        useContext(GoogleContext);
+
+    const importFile = async () => {
+        if (files?.[0]?.id) handleRemoveFile(files[0].id);
+        await handleSelectFile();
```

Comment:
So what happens if this fails? The error is caught somewhere else?

Reply:
is caught in handleSelectFile and sets global Error